org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/folding/AbstractFoldingRangeAcceptor.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2021 TypeFox GmbH (http://www.typefox.io) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.xtext.ide.editor.folding;
+
+import org.eclipse.xtext.util.ITextRegion;
+
+/**
+ * @author Mark Sujew - Initial contribution and API
+ */
+public abstract class AbstractFoldingRangeAcceptor implements IFoldingRangeAcceptor {
+
+	@Override
+	public void accept(int offset, int length) {
+		accept(offset, length, false);
```

Comment:
Can these all be default methods on the interface?

Reply:
You're right. I removed the abstract acceptor.